tests/MakeEnumCommandTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+use Illuminate\Support\Facades\File;
+use function Pest\Laravel\artisan;
+use function Orchestra\Testbench\workbench_path;
+
+it('can create an enum', function () {
+
+
+    /** @noinspection PhpFullyQualifiedNameUsageInspection */
+    if (!class_exists(\Illuminate\Foundation\Console\EnumMakeCommand::class)) return;
+
+    if (File::exists(workbench_path('app/Enums/TestEnum.php'))) {
+        File::delete(workbench_path('app/Enums/TestEnum.php'));
+    }
+    artisan('make:enum TestEnum -s')
+        ->execute();
+
+
+
+
+
+    expect(workbench_path('app/Enums/TestEnum.php'))->toBeFile();
+});
+
+it('can make pure enum', function () {
+
+    /** @noinspection PhpFullyQualifiedNameUsageInspection */
+    if (!class_exists(\Illuminate\Foundation\Console\EnumMakeCommand::class)) return;
```

Comment:
Rather than doing this, skip the tests properly.

```
it(...)
->skip(!class_exists(\Illuminate\Foundation\Console\EnumMakeCommand::class), 'Laravel 11 only')